paasta_tools/cli/cmds/logs.py

```diff
@@ -251,8 +251,10 @@ def paasta_app_output_passes_filter(
     except ValueError:
         log.debug('Trouble parsing line as json. Skipping. Line: %r' % line)
         return False
-
-    timestamp = isodate.parse_datetime(parsed_line.get('timestamp'))
+    try:
+        timestamp = isodate.parse_datetime(parsed_line.get('timestamp'))
+    except ValueError:
```

Comment:
Can you please leave a comment with the link to the github issue.
Also, should it be False? Isn't the timestamp technically in range?

Reply:
I linked the issue. You are right that the timestamp IS in range. However, because the timestamp is not correctly parsed (due to the bug), it cannot proceed to component and cluster checks to give a definitive True. I think false positive is better than missing a log line. So changed to return True here.